run/batch_applier_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/utilitywarehouse/kube-applier/kube"
 	"github.com/utilitywarehouse/kube-applier/log"
 	"github.com/utilitywarehouse/kube-applier/metrics"
+	"github.com/utilitywarehouse/kube-applier/mock_kube"
```

Comment:
I disagree a bit here in terms of style, as it becomes different from mocks under `run` for example.
I would rather create a mock package under `kube` dir and go like:
```
kube_mock  github.com/utilitywarehouse/kube-applier/kube/mock
```
if I needed to split client from mock functions, but of course that is just personal preference

Reply:
I decided to go with what gomock considers canonical:

> -package: The package to use for the resulting mock class source code. If you don't set this, the package name is mock_ concatenated with the package of the input file.

from: https://github.com/golang/mock

I don't see the need to overwrite it and deviate from the default here. Although I do agree consistency is good, happy to update the rest of the packages to the same model?

Reply:
that would be nice, but we can also handle it in a separate card if it is too much work and you want to merge this one faster.